frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "src/components/Button.module.css";
+
+export interface ButtonProps extends React.ComponentProps<"button"> {
+  label: string;
+  kind?: "primary" | "secondary";
+}
```

Comment:
i think it'd be beneficial to explain why we have to use interfaces when passing props when using ts with react

Reply:
I assume you mean why we're required to declare the types of the props, which is because TypeScript says so